CHANGELOG.md

```diff
@@ -1,9 +1,25 @@
 # Unreleased
 
+# 9.13.0 (Feature release)
+
+## New features
```

Comment:
We'll also need to include 'Update to GOV.UK Frontend v3.12' in the new features section – the PR for that will come later, just before we do the release.

Normally we just include it as a bullet item ([like in 9.12](https://github.com/alphagov/govuk-prototype-kit/blob/master/CHANGELOG.md#9120-feature-release)) but I'm not sure how that would work when we have an h3 with copy like we do here.